src/Okta.Sdk/User.cs

```diff
@@ -39,8 +40,8 @@ public Task<IResetPasswordToken> ResetPasswordAsync(bool? sendEmail = true, Canc
             => GetClient().Users.ResetPasswordAsync(Id, sendEmail, cancellationToken);
 
         /// <inheritdoc/>
-        public Task DeactivateOrDeleteAsync(CancellationToken cancellationToken = default(CancellationToken))
-            => GetClient().Users.DeactivateOrDeleteUserAsync(Id, cancellationToken);
+        public Task DeactivateOrDeleteAsync(bool? sendEmail = false, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
We need an overload here so this isn't a breaking change. (Unless I missed where the overload is)